docs/general/post-install/networking/9_advanced/traefik.md

```diff
@@ -22,22 +22,215 @@ Your router must local forward to the host running Traefik on port 80 (only if T
 ### Create Traefik static file
 Every changes to this file require a Traefik restart.
 
-```yml
+e.g. in /mnt/docker-volumes/traefik/traefik.yml
 
+```yml
+# Api and, additionally, a monitoring dashboard (checkout docs for more,
+# disable in production if not required!).
+global:
+  checkNewVersion: true
+  sendAnonymousUsage: true
+
+api:
+  dashboard: true
+  insecure: true
+
+log:
+  level: INFO
+  filePath: "/logs/traefik.log"
+
+accessLog:
+  filePath: "/logs/access.log"
+  format: json
+  filters:
+    statusCodes:
+      - "200-299" # log successful http requests
+      - "400-599" # log failed http requests
+  # collect logs as in-memory buffer before writing into log file
+  bufferingSize: 0
+  fields:
+    headers:
+      defaultMode: drop # drop all headers per default
+      names:
+          User-Agent: keep # log user agent strings
+
+# Entrypoints (ports) Traefik should listen to; here we define two: "http"
+# for unencrypted traffic, and "https" for SSL-encrypted traffic (port 443).
+entryPoints:
+  http:
+    address: ":80"
+    forwardedHeaders:
+      trustedIPs: &trustedIps
+        # Start of Cloudlare's public IP list
```

Comment:
Please consult our rules on breaking the TOS of other services.
Tunneling jellyfin via CF is not permitted and we dont aid in breaking TOS

Reply:
I will document how to exclude in cloudflare the Edge caching, since 2 years I run it like this and it is not against the TOS if properly configured

Reply:
if __any__ of jellyfin video traffic touches thier servers, its against their TOS. We have had users that got banned running JF though CF, i am sory but this is not up for discussion. If you dont tunnel though their servers you dont need to trust their headers. I am telling you plainly, any PR aiding in that will never get merged.